src/pages/academy/components/_side-subscription-banner.js

```diff
@@ -375,7 +375,7 @@ const ArticleEmailBanner = () => {
                                         components={[
                                             <StyledLocalizedLink
                                                 key={0}
-                                                type="tnc/security-and-privacy.pdf"
+                                                to="tnc/security-and-privacy.pdf"
```

Comment:
This is gonna append that route onto the existing route and will not work.

Reply:
hey, using type adds a trailing slash and the pdf doesn't load

Reply:
Using to like this won't work as well. It will add `tnc/security-and-privacy.pdf` to your current URL.